src/Skia/Avalonia.Skia/WriteableBitmapImpl.cs

```diff
@@ -96,9 +96,9 @@ public WriteableBitmapImpl(PixelSize size, Vector dpi, PixelFormat format, Alpha
 
             SKColorType colorType = format.ToSkColorType();
             SKAlphaType alphaType = alphaFormat.ToSkAlphaType();
-            
-            var runtimePlatform = AvaloniaLocator.Current?.GetService<IRuntimePlatform>();
-            
+
+            var runtimePlatform = AvaloniaLocator.Current.GetRequiredService<IRuntimePlatform>();
```

Comment:
This code assumed that IRuntimePlatform might not be registered, so condition below can be removed now, as it is never null.

Currently IRuntimePlatform is always registered regardless of the backend, so that's safe.